src/SDDPoptimize.jl

```diff
@@ -86,21 +92,26 @@ function run_SDDP!(model::SPModel,
         noise_scenarios = simulate_scenarios(model.noises, param.forwardPassNumber)
 
         # Forward pass
-        costs, stockTrajectories, _ = forward_simulations(model,
+        costs, stockTrajectories, _, callSolver = forward_simulations(model,
                             param,
                             V,
                             problems,
                             noise_scenarios)
 
+        #Update the number of call
+        countCallSolver = countCallSolver + callSolver
+
         # Backward pass
-        backward_pass!(model,
+        callSolver = backward_pass!(model,
                       param,
                       V,
                       problems,
                       stockTrajectories,
                       model.noises,
                       false)
 
+        countCallSolver = countCallSolver + callSolver
```

Comment:
I think we should avoid to repeat ourself. What do you think of this: 

countCallSolver += callSolver_forward + callSolver_backward